Restore Qt 4.x support because Qt 5.x is not in Portage yet and 2.82 is a good stabilization target, see:

http://bugs.gentoo.org/485038

--- qt/qtr.pro
+++ qt/qtr.pro
@@ -12,9 +12,13 @@
 man.files = transmission-qt.1
 
 CONFIG += qt thread debug link_pkgconfig
-QT += network dbus widgets
+QT += network dbus
 PKGCONFIG = fontconfig libcurl openssl libevent
 
+greaterThan(QT_MAJOR_VERSION, 4) {
+    QT += widgets
+}
+
 TRANSMISSION_TOP = ..
 
 include(config.pri) 
--- qt/favicon.cc
+++ qt/favicon.cc
@@ -14,8 +14,13 @@
 #include <QNetworkAccessManager>
 #include <QNetworkReply>
 #include <QNetworkRequest>
-#include <QStandardPaths>
 
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
+ #include <QDesktopServices>
+#else
+ #include <QStandardPaths>
+#endif
+
 #include "favicon.h"
 
 /***
@@ -40,7 +45,13 @@
 QString
 Favicons :: getCacheDir( )
 {
-    const QString base = QStandardPaths::writableLocation (QStandardPaths::CacheLocation);
+    const QString base =
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
+        QDesktopServices::storageLocation( QDesktopServices::CacheLocation );
+#else
+        QStandardPaths::writableLocation( QStandardPaths::CacheLocation );
+#endif
+
     return QDir( base ).absoluteFilePath( "favicons" );
 }
 
--- qt/file-tree.cc
+++ qt/file-tree.cc
@@ -885,7 +885,12 @@
   for (int i=0; i<NUM_COLUMNS; ++i)
     {
       setColumnHidden (i, (i<FIRST_VISIBLE_COLUMN) || (LAST_VISIBLE_COLUMN<i));
+
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
+      header()->setResizeMode(i, QHeaderView::Interactive);
+#else
       header()->setSectionResizeMode(i, QHeaderView::Interactive);
+#endif
     }
 
   connect (this, SIGNAL(clicked(const QModelIndex&)),

------------------------------------------------------------------------
